src/Microsoft.AspNet.Diagnostics/Views/BaseView.cs

```diff
@@ -55,7 +55,7 @@ public abstract class BaseView
             Output = new StreamWriter(Response.Body);
```

Comment:
How about using the StreamWriter constructor with the leaveOpen parameter? Then you can dispose the StreamWriter.